styleguide/app/index.html

```diff
@@ -196,6 +207,78 @@ <h2 class="sidebar-content-heading">Components</h2>
                                 <div class="slider"></div>
                             </div>
                         </section>
+                        <section>
+                            <h3>Range slider</h3>
+                            <div>
```

Comment:
I think this element needs `class="slider-container"` to get teh handles to light up correctly.

![screenshot from 2017-02-21 17 39 17](https://cloud.githubusercontent.com/assets/1014341/23188433/aeda55de-f85c-11e6-82ad-9781c406f287.png)